src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,87 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
 public class StringManipulation implements StringManipulationInterface {
 
+    String s;
     @Override
     public String getString() {
-        return null;
```

Comment:
This is methods breaks and the return method never gets accessed due to always returning null.